src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IFieldReferenceExpression.cs

```diff
@@ -241,5 +241,69 @@ void M()
 
             VerifyOperationTreeAndDiagnosticsForTest<MemberAccessExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics);
         }
+
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void IFieldReference_StaticCallWithInstanceReceiver()
```

Comment:
Typo: static member access?

Reply:
I don't think so? I'm accessing a static field off of an instance in this test, so that's what I named it.

---
In reply to: [148816410](https://github.com/dotnet/roslyn/pull/22972#discussion_r148816410) [](ancestors = 148816410)